docs/resources/provider.md

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_provider Resource
+
+Within a metastore, Unity Catalog provides the ability to create a provider which contains a list of shares that have been shared with you.
+
+A `databricks_provider` is contained within [databricks_metastore](metastore.md) and can contain a list of shares that have been shared with you.
+
+Note that Databricks to Databricks sharing automatically creates the provider.
+
+## Example Usage
+
+```hcl
+resource "databricks_provider" "dbprovider" {
+  name = "sri-terraform-test-provider"
+  comment = "made by terraform 2"
+  authentication_type = "TOKEN"
+  recipient_profile_str = jsonencode(
+    {
+        "shareCredentialsVersion":1,
+        "bearerToken":"token",
+        "endpoint":"endpoint",
```

Comment:
is the endpoint output from any other terraform resource?

Reply:
it wont be because the "provider" will be creating the recipient object. So in theory you get all this info when you go to a certain website and download the credentials file.